src/web/services/feedback-sessions.service.ts

```diff
@@ -178,6 +178,26 @@ export class FeedbackSessionsService {
     return this.httpRequestService.get(ResourceEndpoints.SESSIONS, paramMap);
   }
 
+  /**
+   * Gets all sessions for the student by calling API.
+   */
+  getFeedbackSessionsForStudentByAdmin(courseId?: string): Observable<FeedbackSessions> {
```

Comment:
Can we combine this function with the function above by adding an additional parameter?

Reply:
Done that!